.prettierrc

```diff
@@ -1,6 +1,5 @@
 {
   "semi": false,
   "singleQuote": true,
-  "bracketSpacing": false,
```

Comment:
This rule will make editing the last attribute more difficult 🤔

Generally if I want to add a new attribute, I will move the cursor to the last attribute line and add a new line, or delete this line for deleting the last attribute. But with bracketSameLine, you have to handle `>` manually.

Reply:
ok, no problem.